crates/bevy_ecs/src/schedule/stage.rs

```diff
@@ -45,7 +51,190 @@ impl_downcast!(Stage);
+    "bevy_dylib",
+    "bevy_dynamic_plugin",
+    "bevy_ecs",
+    "bevy_gilrs",
+    "bevy_gltf",
+    "bevy_input",
+    "bevy_internal",
+    "bevy_log",
+    "bevy_macro_utils",
+    "bevy_math",
+    "bevy_pbr",
+    "bevy_reflect",
+    "bevy_render",
+    "bevy_scene",
+    "bevy_sprite",
+    "bevy_tasks",
+    "bevy_text",
+    "bevy_transform",
+    "bevy_ui",
+    "bevy_utils",
+    "bevy_wgpu",
+    "bevy_window",
+    "bevy_winit",
+    "bevy_ai",
+    "bevy_android",
+    "bevy_animation",
+    "bevy_async",
+    "bevy_base",
+    "bevy_bounds",
+    "bevy_box2d",
+    "bevy_bullet",
+    "bevy_camera",
+    "bevy_cli",
+    "bevy_collision",
+    "bevy_controller",
+    "bevy_csharp",
+    "bevy_curve",
+    "bevy_demo",
+    "bevy_editor",
+    "bevy_event",
+    "bevy_executor",
+    "bevy_extra",
+    "bevy_game",
+    "bevy_geometry",
+    "bevy_gizmo",
+    "bevy_gizmos",
+    "bevy_gpu",
+    "bevy_ik",
+    "bevy_imgui",
+    "bevy_ios",
+    "bevy_jobs",
+    "bevy_joystick",
+    "bevy_js",
+    "bevy_kernel",
+    "bevy_library",
+    "bevy_lighting",
+    "bevy_log",
+    "bevy_lua",
+    "bevy_marketplace",
+    "bevy_navmesh",
+    "bevy_net",
+    "bevy_node_graph",
+    "bevy_nodes",
+    "bevy_nphysics",
+    "bevy_particles",
+    "bevy_pathing",
+    "bevy_physics",
+    "bevy_physx",
+    "bevy_picking",
+    "bevy_playstation",
+    "bevy_prefab",
+    "bevy_project",
+    "bevy_raycast",
+    "bevy_raytracing",
+    "bevy_rustgpu",
+    "bevy_script",
+    "bevy_scripting",
+    "bevy_sdf",
+    "bevy_sdl2",
+    "bevy_shape",
+    "bevy_shapes",
+    "bevy_skeleton",
+    "bevy_spirv",
+    "bevy_store",
+    "bevy_streaming",
+    "bevy_switch",
+    "bevy_system",
+    "bevy_test",
+    "bevy_thread",
+    "bevy_tile",
+    "bevy_tiled",
+    "bevy_tools",
+    "bevy_util",
+    "bevy_vfx",
+    "bevy_visibility",
+    "bevy_voxel",
+    "bevy_vr",
+    "bevy_web",
+    "bevy_websocket",
+    "bevy_widget",
+    "bevy_widgets",
+    "bevy_xbox",
+    "bevy_xr",
+];
+
+impl ReportExecutionOrderAmbiguities {
+    /// Disables all messages reported by the ambiguity checker.
+    pub fn off() -> Self {
+        Self {
+            level: AmbiguityReportLevel::Off,
+            ..Default::default()
+        }
+    }
+
+    /// Displays only the number of unresolved ambiguities detected by the ambiguity checker. This is the default behavior.
+    pub fn minimal() -> Self {
+        Self {
+            level: AmbiguityReportLevel::Minimal,
+            ..Default::default()
+        }
+    }
+
+    /// Displays a full report of ambiguities detected by the ambiguity checker.
+    pub fn verbose() -> Self {
+        Self {
+            level: AmbiguityReportLevel::Verbose,
+            ..Default::default()
+        }
+    }
+
+    /// Adds the given crates to be ignored by ambiguity checker. Check [`ReportExecutionOrderAmbiguities`] for more details.
+    pub fn ignore(mut self, create_prefix: &[&str]) -> Self {
```

Comment:
I think this should take a single string instead of a list. We could also have an "ignore_all" function that takes an iterator.

Reply:
Done. Renamed this one to `ignore_all` and added a new one `ignore` which just takes a single `&str`